From 3fe9c23ac2c8cf4b3dbf34d83cd5061cc8a3b406 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <yasufum.o@gmail.com>
Date: Thu, 25 Jul 2019 21:04:02 +0900
Subject: [PATCH] shared/sec: move defines of port types

This patch is to move following defines to be able to be used from
spp_pcap.

  #define SPPWK_PHY_STR "phy"
  #define SPPWK_VHOST_STR "vhost"
  #define SPPWK_RING_STR "ring"

Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 src/shared/secondary/spp_worker_th/cmd_utils.h  | 5 -----
 src/shared/secondary/spp_worker_th/data_types.h | 5 +++++
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/shared/secondary/spp_worker_th/cmd_utils.h b/src/shared/secondary/spp_worker_th/cmd_utils.h
index 56f11ac..ecc1141 100644
--- a/src/shared/secondary/spp_worker_th/cmd_utils.h
+++ b/src/shared/secondary/spp_worker_th/cmd_utils.h
@@ -27,11 +27,6 @@
 #define SPPWK_TYPE_PCAP_STR "pcap"
 #define SPPWK_TYPE_NONE_STR "unuse"
 
-/** Identifier string for each interface */
-#define SPPWK_PHY_STR "phy"
-#define SPPWK_VHOST_STR "vhost"
-#define SPPWK_RING_STR "ring"
-
 /** Waiting time for checking update (not used for spp_pcap). */
 #define SPP_CHANGE_UPDATE_INTERVAL 10  /* micro sec */
 
diff --git a/src/shared/secondary/spp_worker_th/data_types.h b/src/shared/secondary/spp_worker_th/data_types.h
index 7b24df3..f6dd60f 100644
--- a/src/shared/secondary/spp_worker_th/data_types.h
+++ b/src/shared/secondary/spp_worker_th/data_types.h
@@ -10,6 +10,11 @@
 #define STR_LEN_SHORT 32  /* Size of short string. */
 #define STR_LEN_NAME 128  /* Size of string for names. */
 
+/** Identifier string for each interface */
+#define SPPWK_PHY_STR "phy"
+#define SPPWK_VHOST_STR "vhost"
+#define SPPWK_RING_STR "ring"
+
 /* TODO(yasufum) confirm usage of this value and why it is 4. */
 #define PORT_CAPABL_MAX 4  /* Max num of port abilities. */
 
-- 
2.17.1

